README.md

```diff
@@ -71,3 +71,21 @@ Once you have set some variables, you can use them in graph queries: either for
 ## Installing from sources
 
 Additional information on installing from sources can be found on [hawkular.org](http://www.hawkular.org/hawkular-clients/grafana/docs/quickstart-guide/).
+
+## Troubleshooting
+
+### Grafana fails to establish a connection or get data from hawkular
+
+* Check the URL: `[host]/hawkular/metrics`. Make sure there's no ending slash. When you open up this URL in a browser you should see the Hawkular logo, the installed version and a mention that the service is started.
+
+* Make sure any credential / token matches your installation. In general, if you installed a standalone hawkular-metrics server without any specific configuration you probably don't have any authentication information to provide. If you installed hawkular-services using its [installation guide](http://www.hawkular.org/hawkular-services/docs/installation-guide/) you will probably have to fill-in the basic auth fields. If you are using Hawkular from OpenShift, you have to provide a Bearer token in the `Token` field. Tokens can be [generated temporarily](https://docs.openshift.com/enterprise/3.1/architecture/additional_concepts/authentication.html) (got to `[OpenShift host]/oauth/token/request`) or from a [Service account](https://docs.openshift.com/container-platform/3.3/rest_api/index.html#rest-api-serviceaccount-tokens) in OpenShift.
+
+* Check your browser's javascript debugging tool. If you see an error mentioning issues with CORS, switch to `proxy` mode in the datasource configuration.
```

Comment:
It's what is recommended in our doc, but the default is direct and we don't have control on that. There's been several people doing this mistake recently, so it worth saying that again on more time :)